lib/diagnostic.rb

```diff
@@ -108,10 +108,12 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = Hash.new
+person[:favorite_number] = [7]
+person[:first_name] = ["rowland"]'
 
```

Comment:
To make this one line long you could do 
`'person = {first_name: "Rowland", favorite_number: 7}'`